benchmarks/cluster/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+add_library(cluster_benchmark_helpers
+  data.cpp
+  print_timers.cpp
+)
+target_compile_features(cluster_benchmark_helpers PRIVATE cxx_std_17)
```

Comment:
don't we get this transitively from the ArborX target?

Reply:
We do. I'll remove it. I added it before I linked against ArborX, and forgot to remove it.